ph5/utilities/tests/test_pforma_io.py

```diff
@@ -0,0 +1,113 @@
+'''
+Tests for pforma_io
+'''
+import unittest
+import logging
+from StringIO import StringIO
+from contextlib import contextmanager
+import os
+import sys
+import shutil
+import tempfile
+import ph5
+from ph5.utilities import segd2ph5, pforma_io, tabletokef
+
+
+@contextmanager
+def captured_output():
+    new_out, new_err = StringIO(), StringIO()
+    old_out, old_err = sys.stdout, sys.stderr
+    try:
+        sys.stdout, sys.stderr = new_out, new_err
+        yield sys.stdout, sys.stderr
+    finally:
+        sys.stdout, sys.stderr = old_out, old_err
+
```

Comment:
This is great.
I think this contextmanager will be useful in a lot of tests. Lets put this somewhere it can be imported and reused.

Reply:
Is anything printed here that is not printed through the logger?

Reply:
Yes. There are some messages originally printed out from pfoma_io which is not through logger. Do you want me to try to print it through logger instead?

Reply:
Example of messages printed out from pfoma_io:
A
Hard link miniPH5_00001.ph5 to Sigma, preserve A/master.ph5.
B
C
D